tendrl/monitoring_integration/webhook/webhook_receiver.py

```diff
@@ -1,85 +1,69 @@
-import json
+import cherrypy
+from flask import Flask
+from flask import request
+from paste.translogger import TransLogger
 import socket
 import threading
 
-from werkzeug.serving import run_simple
-from werkzeug.wrappers import Response
-
 from tendrl.commons.event import Event
 from tendrl.commons.message import ExceptionMessage
 from tendrl.commons.utils import log_utils as logger
 from tendrl.monitoring_integration.alert.handlers import AlertHandlerManager
 
 PORT = 8789
+app = Flask(__name__)
 
 
 class WebhookReceiver(threading.Thread):
     def __init__(self):
         super(WebhookReceiver, self).__init__()
         self.daemon = True
-        self.alert_handler = AlertHandlerManager()
 
-    def _application(self, env, start_response):
+    @app.route('/grafana_callback', methods=["POST"])
+    def _application():
         try:
-            if env['PATH_INFO'] != '/grafana_callback':
-                response = Response('Alert not found')
-                response.headers['content-length'] = len(response.data)
-                response.status_code = 404
+            alert_handler = AlertHandlerManager()
+            data = request.json
+            if "ruleId" in data:
+                alert_handler.handle_alert(
+                    data["ruleId"]
+                )
```

Comment:
Dont we need to set response here?

Reply:
Returning ok from this function is enough we don't need to response